webservice/models/webservice_backend.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class WebserviceBackend(models.Model):
+
+    _name = "webservice.backend"
+    _inherit = ["collection.base", "server.env.mixin"]
+    _description = "WebService Backend"
+    _server_env_section_name_field = "code"
```

Comment:
you can use https://github.com/OCA/server-env/blob/13.0/server_environment/models/server_env_tech_name_mixin.py